### ISSUE TITLE
Bug: Characters in old readme files can cause problems

### ISSUE THREAD

**Issue by zoidy** [UAL-ODIS/LD-Cool-P#220]
**Describe the bug**
When old readme files are created (e.g., after running update_readme), the files are named with non-standard characters which can cause problems with some applications. E.g., the built-in Windows zip function complains that the files contain names with characters that are not supported and aborts the zipping process. It would be better to use only standard ASCII characters for file names. This is a minor issue but I'm adding it for documentation purposes 

**Reproduction Steps**
Zip a folder with old readme files using the built-in functionality in Windows Explorer.

**Outputted Messages**
None

**Version information**
- All versions

*Expected behavior*
Zipping files works

*Screenshots*
![image](https://user-images.githubusercontent.com/7082025/121438767-75b44780-c939-11eb-966a-a1483d1e397c.png)

*Additional context*

**Implemented in**: TBD

**Comment by astrochun** [comment]
I'm really confused. The characters used are hyphens and underscores. If you plan to zip up old README this really should be done on the server not through SSH mounting.

**Comment by zoidy** [comment]
I'm not doing it via SSH mount, though I figured out the problem. I looked at it on the server and it looks like you're using colon as the separator for the timestamp.  That is an invalid character for file names in Windows so zipping it up on the server and attempting to unzip on Windows would actually cause bigger problems. It looks like the Windows SSHFS mount was actually preventing a bigger problem by replacing the invalid character with something that wouldn't completely break the system. The fix here is to use characters that are valid on all operating systems.
